### ISSUE TITLE
NVDA fails to read the list of suggestions when entering a formula in a cell in Excel

### ISSUE THREAD

**Issue by k-kolev1985** [nvaccess/nvda#10542]
### Steps to reproduce:
1. Launch NVDA.
2. Open a workbook in Excel (an empty one should suffice).
3. Press F2 to enter editing mode for the focused cell.
4. Start entering a formula. I type "=s". A list of suggestions beginning with "s" appear below the cell.
5. Use the up/down arrow keys to navigate between the suggestions.
6. Observe how the focus moves between the suggestions in the list and what NVDA reports for each focus change.

### Actual behavior:
NVDA reports only the content (e.g. "=s") of the edited cell on each focus change in the list of suggestions.

### Expected behavior:
NVDA should read each focused item in the list of suggestions when the item gains focus.

### System configuration
- Operating system: Windows 10 Pro version 1909 (build 18363.476), 64-bit, in Bulgarian with all locale settings set to "Bulgarian".
- NVDA version: alpha-19266,67bf1df6, in bulgarian.
- Microsoft Excel version: Microsoft Excel for Office 365, 32-bit, MSO (16.0.12130.20232), click to launch, in bulgarian (information taken from the "About Excel" dialog).
- Processor: Intel Core i5-9400F at 2.90GHz.
- RAM Memory: 16.00GB.
- Graphics: MSI GeForce GTX-1050TI Gaming X 4G, 4096MB dedicated memory + 4065MB shared system memory, desktop resolution set to 1920x1080.
- Sound Card: Realtek ALC892 at Intel Cannon Point PCH.

#### Does the issue still occur after restarting your PC?
Yes - it does.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes - v2019.2.1 - the behavior is the same.

**Comment by CyrilleB79** [comment]
This issue has also been reported today on a French mailing list: https://groups.io/g/nvda-fr/topic/79024793#1111

Also confirmed on my side with NVDA 2020.4beta1.

**Comment by Adriani90** [comment]
Closing as duplicate of #6786.
